src/config/dynamic-theme-fixes.config

```diff
@@ -2836,6 +2836,15 @@ button
 
 ================================
 
+comicfury.com
+
+CSS
+html {
+    background-color: revert !important;
+}
```

Comment:
Same thing as in your previous PR.

Reply:
Ok, I did some experimenting to figure out another solution. It looks like setting the background color of `#wrapper` to `inherit` also solves the core issue. The only issue is the top margin on the `#footer`, but replacing it with a bottom padding on `#wrapper` fixes that.